jujugui/static/gui/src/app/app.js

```diff
@@ -2067,8 +2067,8 @@ YUI.add('juju-gui', function(Y) {
      *
      */
     authorizeCookieUse: function(req, res, next) {
-      var ga_key = this.get('GA_key');
-      if (ga_key) {
+      var GTM_enabled = this.get('GTM_enabled');
```

Comment:
I'm not sure this is going to work--GTM_enabled isn't an attr on app, so this.get shouldn't work.

That said GA_key wasn't an attr either--I'm not sure how this worked previously. @hatched, you have any ideas?